src/Console/Application.php

```diff
@@ -86,6 +92,9 @@ public function doRun(InputInterface $input, OutputInterface $output): int
             : $currentWorkingDirectory.DIRECTORY_SEPARATOR.'.deptrac.cache';
 
         $factory = new ServiceContainerBuilder($currentWorkingDirectory);
+        if (false === $input->hasParameterOption('--clear-cache', true)) {
```

Comment:
This looks weird. Shouldn't we check if the option is set, so `true` instead?

Reply:
You were absolutely right, fixed.